simapp/v2/simdv2/cmd/testnet_test.go

```diff
@@ -0,0 +1,27 @@
+package cmd_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"cosmossdk.io/core/transaction"
+	"cosmossdk.io/simapp/v2"
+	"cosmossdk.io/simapp/v2/simdv2/cmd"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	svrcmd "github.com/cosmos/cosmos-sdk/server/cmd"
```

Comment:
we have the server/v2 equivalent of this

Reply:
Oh, okay. I just copied the same code from root_test.go. Will update both tests.